src/content/en/fundamentals/performance/http2/index.md

```diff
@@ -0,0 +1,236 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: HTTP/2 will make our applications faster, simpler, and more robust
+
+{# wf_updated_on: 2016-09-29 #}
+{# wf_published_on: 2016-09-29 #}
+
+# Introduction to HTTP/2
```

Comment:
Missing ` {: .page-title }`, which is causing the build to fail. All pages need a page title. Should be in the format of: `# Introduction to HTTP/2  {: .page-title }`